e2e/tests/transfer/base_test.go

```diff
@@ -274,12 +283,17 @@ func (s *TransferTestSuite) TestSendEnabledParam() {
 	})
 
 	t.Run("change send enabled parameter to disabled", func(t *testing.T) {
-		changes := []paramsproposaltypes.ParamChange{
-			paramsproposaltypes.NewParamChange(transfertypes.StoreKey, string(transfertypes.KeySendEnabled), "false"),
-		}
+		if isSelfManagingParams {
```

Comment:
You should be able to use a [`FeatureReleaases`](https://github.com/cosmos/ibc-go/blob/main/e2e/semverutil/semver.go#L10) matrix to do custom versioning here. 

Here is an example of how its [defined](https://github.com/cosmos/ibc-go/blob/main/e2e/tests/transfer/base_test.go#L400-L409) and [used in a test](https://github.com/cosmos/ibc-go/blob/main/e2e/tests/transfer/base_test.go#L442-L447)

Reply:
implemented